Readme.md

```diff
@@ -10,18 +10,18 @@
 Are you looking to get started? [This is the guide](https://github.com/internetarchive/openlibrary/blob/master/CONTRIBUTING.md) you are looking for. You may wish to learn more about [Google Summer of Code (GSoC)?](https://github.com/internetarchive/openlibrary/wiki/Google-Summer-of-Code) or [Hacktoberfest](https://github.com/internetarchive/openlibrary/wiki/Hacktoberfest).
 
 ## Table of Contents
-   - [Overview](#overview)
-   - [Installation](#installation)
-   - [Code Organization](#code-organization)
-   - [Architecture](#architecture)
-     - [The Frontend](https://github.com/internetarchive/openlibrary/wiki/Frontend-Guide)
-     - [The Backend](#the-backend)
-     - [The Service Architecture](https://github.com/internetarchive/openlibrary/wiki/Production-Service-Architecture)
-   - [Developer's Guide](#developers-guide)
-   - [Running Tests](#running-tests)
-   - [Contributing](CONTRIBUTING.md)
-   - [Public APIs](https://openlibrary.org/developers/api)
-   - [FAQs](https://openlibrary.org/help/faq)
+- [Open Library](#open-library)
```

Comment:
I think unless you want to make a strong case for it lets not change the table of contents.

Reply:
It is not me, but the plugIn I use for vs-code, because the table content is generated by plugin, I think it isn’t good order

Reply:
<img width="442" alt="image" src="https://github.com/internetarchive/openlibrary/assets/15892424/f0d650e6-1a50-4d41-8e23-6ced685a0e5b">

Reply:
I delete these change